### ISSUE TITLE
Screen readers do not announce TreeView node's checkbox presence and state on focusing the respective node

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#3964]
### Bug report
Allow screen readers to announce the presence of a TreeView node checkbox. Announce the checkbox state while the respective node is focused.

### Reproduction of the problem
* Enable any screenreader. NVDA for Firefox, for example;
* Go to the [Checkboxes demo](https://demos.telerik.com/kendo-ui/treeview/checkboxes);
* Navigate from one node to another;
* The node content and its selected state are announced, but the presence of the checkbox is not;
* Press spacebar - the state of the checkbox will change, but the reader will announce only the pressed key name;

**Comment by veselints** [comment]
also reported in Ticket ID: 1362898

**Comment by veselints** [comment]
The following workarownd announces the checked state of the checkbox:
[Dojo Sample Here](https://dojo.telerik.com/ESOcUboN/8)

Tested with NVDA in Firefox
